drop-down/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@antol/widget-drop-down",
+  "description": "Simple drop-down widget for Grist.",
+  "homePage": "https://github.com/Antol/grist-widget",
+  "version": "0.0.1",
+  "grist": {
+    "name": "Drop-down Widget",
+    "url": "https://antol.github.io/grist-widget/drop-down/index.html",
```

Comment:
Can you change the URL to point to this site?

Reply:
Yes, of course